src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
```

Comment:
apply code styling recommendations from a task checklist
```suggestion
        <a href="#home" class="header__logo">
          <img src="./images/logo.png" alt="logo">
        </a>
```